dune

```
@@ -1 +1 @@
-(vendored_dirs vendors)
+(vendored_dirs vendors irmin)
```

```suggestion
(vendored_dirs vendors)
```

Same idea as the `.gitignore` comment.